src/main/java/org/icgc/argo/program_service/converter/ProgramConverter.java

```diff
@@ -146,6 +146,7 @@ default ProgramDetails ProgramEntityToProgramDetails(ProgramEntity value) {
   @Mapping(target = "unknownFields", ignore = true)
   @Mapping(target = "mergeUnknownFields", ignore = true)
   @Mapping(target = "allFields", ignore = true)
+  @Mapping(target = "roleValue", ignore = true)
```

Comment:
why ignore this? Mapstruct will take care of the conversion from EgoUser.role to User.role. All you have to do is the following:
1. define the boxing method for converting a `UserRole` to `UserRoleValue` in the converter interface
```java
  default UserRoleValue boxUserRole(@NonNull UserRole role){
	  return UserRoleValue.newBuilder().setValue(role).build();
  }
```
2. Change `User` proto definition to use `UserRoleValue` instead of `UserRole`
3. add ` @Mapping(target = "mergeRole", ignore = true)` annotation to the `egoUserToUser` method